packages/api-runner/lib/fetchers/getPoolChainData.js

```diff
@@ -78,7 +78,7 @@ export const getPoolChainData = async (chainId, poolGraphData, fetch) => {
         .prizePeriodStartedAt()
         .prizePeriodRemainingSeconds()
         .prizePeriodSeconds()
-        .estimateRemainingBlocksToPrize(ethers.utils.parseEther('14'))
+        .estimateRemainingBlocksToPrize(SECONDS_PER_BLOCK[chainId] || SECONDS_PER_BLOCK[1])
```

Comment:
Would be good if 1 was `NETWORK.mainnet` here, if we used a data structure to map chainIds to a more human readable format. Maybe even the one in `evm-chains-extended`